charts/prefect-server/values.schema.json

```diff
@@ -313,6 +313,16 @@
               "type": "integer",
               "title": "FS Group",
               "description": "set server pod's security context fsGroup"
+            },
+            "seccompProfile": {
+              "type": "object",
```

Comment:
is `additionalProperties` defaulting to true here (eg. if you need to pass `localhostProfile`)? or do we need to explicitly set it here?

Reply:
Hi @dk-oc - just following up here.

Reply:
Sorry I missed the notification.
As far as I understand it defaults to true if a podSecurityContext is set. 
I use a values.yml as such and can deploy the chart in the restricted environment I am running
```
server:
  containerSecurityContext:
    capabilities:
      drop: ["ALL"]
  podSecurityContext:
    seccompProfile:
      type: RuntimeDefault
```

Reply:
@dk-oc - I believe the question was more about what happens if you set `type: Localhost`, as that one requires an extra field to be set named `localhostProfile`:

```yaml
securityContext:
  seccompProfile:
    type: Localhost
    localhostProfile: my-profiles/profile-allow.json
```

- [K8s docs](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-seccomp-profile-for-a-container)
- [API docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#seccompprofile-v1-core)

We'd need to either set `additionalProperties` here, or explicitly define the `localhostProfile`